libraries/fula-client/src/index.ts

```diff
@@ -1,165 +1,180 @@
 import type {SchemaProtocol} from "@functionland/file-protocol";
```

Comment:
Changes to the library should go in a separate PR/issue from changes to the samples.